cmd/run.go

```diff
@@ -32,6 +32,7 @@ func newRunCmd(kd *driver.KubeDriver, out io.Writer) *cobra.Command {
 		},
 	}
 	addRunDirFlag(cmd, &actor.RunDir)
+	addReuseResult(cmd, &actor.ReuseResult)
```

Comment:
I don't remember if we discussed to add the `reuseResult` flag to both `iter8 k run` and `iter8 run`. I think maybe it should only be for `iter8 k run`? Then I will need to change this.

Reply:
Good for now.